README.md

```diff
@@ -87,15 +87,15 @@ or
 ```
 
 - Pulling your request. Click on the Pull requests tab on the forked github repository.
-  Note : A pull request allows your changes to be merged with the original project.
+  Note: A pull request allows your changes to be merged with the original project.
 
 ```
```

Comment:
```suggestion
```

Reply:
@schmelto you removed the text but forgot to remove the code block, you need to fix this and https://github.com/EddieHubCommunity/hacktoberfest-practice/pull/368#discussion_r659293632